public/js/main.js

```diff
@@ -0,0 +1,96 @@
+/*page preloader*/
+function PreLoader(){
+    setTimeout(()=>{
+        $('.page-loader').addClass('d-none')
+    },1000)
+}
+
+
+
+/* banner sliders*/
+$(document).ready(function () {
+    PreLoader()
+    $(".owl-carousel").owlCarousel();
+});
+
+/*banner slider*/
+$('.slider-wrap').owlCarousel({
+    loop: true,
+    autoplay: true,
+    animateOut: 'fadeOut',
+    margin: 10,
+    nav: false,
+    responsive: {
+        0: {
+            items: 1
+        },
+        600: {
+            items: 1
+        },
+        1000: {
+            items: 1
+        }
+    }
```

Comment:
if this is fix that it will show always one item then remove responsive option and simply add **items: 1** as option

Reply:
Fixed